### PR TITLE
Eager docs

### PR DESCRIPTION
**Pull request by jtrain**
Documents #6

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -293,3 +293,26 @@ Sentry:
...
 
 You could also log to a file by using the built-in ``logging.FileHandler``.
+
+Useful Recipes
+==============
+These recipes aren't officially supported features of `django-postgres-queue`. We provide them so that you can mimick some of the common features in other task queues.
+
+`ALWAYS_EAGER`
+--------------
+Celery uses the `ALWAYS_EAGER` setting to run a task immediately, without queueing it for a worker. It could be used during tests, and while debugging in a development environment with any workers turned off.
```
Comment (gavinwahl) [inline README.rst]: Celery uses `CELERY_ALWAYS_EAGER`

File: README.rst [hunk 2]
```diff
@@ -293,3 +293,26 @@ Sentry:
...
+
+.. code:: python
+
+    class EagerAtLeastOnceQueue(AtLeastOnceQueue):
+        def enqueue(self, *args, **kwargs):
+            job = super().enqueue(*args, **kwargs)
+            if settings.CELERY_ALWAYS_EAGER:
```
Comment (gavinwahl) [inline README.rst]: Let's use a different setting like `QUEUE_ALWAYS_EAGER`

File: README.rst [hunk 3]
```diff
@@ -293,3 +293,26 @@ Sentry:
...
+                self.run_job(job)
+            return job
+
```
Comment (gavinwahl) [inline README.rst]: Remove trailing whitespace

**Comment by gavinwahl** [comment]
This looks good, thank you. Did you try that code to make sure it works?

**Comment by jtrain** [comment]
I run this code in prod at the moment. It's good.

File: README.rst (hunk 2 above)
Comment (jtrain) [reply README.rst]: I originally had DPQ_ALWAYS_EAGER, but that looked like we supported it. CELERY_ALWAYS_EAGER made it clearer that it wasn't our thing, it was their thing we are mimicking. What benefit does QUEUE_ALWAYS_EAGER give? Who is the QUEUE namespace?

Comment (gavinwahl) [reply README.rst]: It's not a namespace it's just saying what is always eager

**Comment by jtrain** [comment]
I've cut the empty lines from the end of the file, and renamed `.*ALWAYS_EAGER` as appropriate.

Regarding the word namespace, that's how I would describe when a library groups their settings with a similar first word. e.g. `CELERY_*` In this case using the `CELERY` namespace. I guess everyone is different and you must refer to this in a different way.

**Comment by gavinwahl** [comment]
Thanks!

**Pull request merged**